tests/driver_adxl345/main.c

```diff
@@ -31,11 +31,14 @@ int main(void)
     adxl345_t dev;
     adxl345_data_t data;
 
+    dev.i2c = ADXL345_PARAM_I2C;
+    dev.addr = ADXL345_PARAM_ADDR;
+
     puts("ADXL345 test application");
     printf("Initializing ADXL345 accelerometer at I2C_DEV(%i)... ",
-           adxl345_params->i2c);
+           dev.i2c);
 
-    if (adxl345_init(&dev, adxl345_params) == ADXL345_OK) {
+    if (adxl345_init(&dev, (adxl345_params_t*)adxl345_params) == ADXL345_OK) {
```

Comment:
@haukepetersen, were you mentioning this regarding the fact only one sensor can be initialized at a time ? Then yes, I admit the line should be:
```
adxl345_init(&dev, &adxl345_params[0]) == ADXL345_OK
```